compiler/compile-config/compile-config-2.01-grounding

```diff
@@ -73,11 +37,12 @@ def withRedefinitionsWithJoinsForIdColumns:
         ],
         style: "cmd_extractor", cmd: "
         : ${DEEPDIVE_GROUNDING_DIR:=\"$DEEPDIVE_APP\"/run/model/grounding}
-        : ${DEEPDIVE_GROUNDING_DENSE_MULTINOMIAL:=false}
         varPath=\"$DEEPDIVE_GROUNDING_DIR\"/variable/\(.variableName | @sh)
         mkdir -p \"$varPath\"
         cd \"$varPath\"
 
+        deepdive sql \"ANALYZE VERBOSE \(.variablesTable)\"
```

Comment:
Postgres-specific.  Use [`db-analyze`](https://github.com/HazyResearch/deepdive/blob/master/database/db-driver/postgresql/db-analyze) instead.  Adding `VERBOSE` to postgres' may be a good idea.